index.html

```diff
@@ -257,13 +257,10 @@ <h2>Contact me</h2>
                 project that needs coding. I’d love to help with it!
             </p>
             <form action="https://formspree.io/f/xgebjlno" method="post">
-                <input type="text" name="fullName" id="" placeholder="Enter Your Fullname" maxlength="30" required>
-                <input type="email" name="email" id="" placeholder="Email" required>
-                <textarea name="message" id="" placeholder="Write your message here..." maxlength="500" required></textarea>
-                <div class="button-container">
+                <input type="text" name="fullName" placeholder="Enter Your Fullname" maxlength="30" required>
+                <input type="email" name="email" placeholder="Email" required>
+                <textarea name="message" placeholder="Write your message here..." maxlength="500" required></textarea>
                 <button type="submit" class="contact-btn">Get in touch</button>
```

Comment:
> - [ ] Based on the request made by the previous code reviewer, it appears that you haven't included the button's hover and pressed states in your implementation. It would be greatly appreciated if you could incorporate the necessary interactions for these states. Doing so will enhance the user experience and contribute to a more engaging website. Keep up the good work!👍